FirebaseFunctions/Sources/Functions.swift

```diff
@@ -57,19 +57,21 @@ internal enum FunctionsConstants {
   // The region to use for all function references.
   internal let region: String
 
+  // MARK: - Public APIs
```

Comment:
Let's chat about how we want to structure files that have a public API surface - maybe creating a `Sources/Public` directory and any file that has at least one public API must go there? WDYT?